CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -1178,22 +1266,17 @@ local function onPurchaseSuccess()
 	local descriptionText = PURCHASE_MSG.SUCCEEDED
 
 	descriptionText = string.gsub(descriptionText, "itemName", string.sub(PurchaseData.ProductInfo["Name"], 1, 20))
-	ItemDescriptionText.Text = descriptionText
+	setItemDescriptionText(descriptionText)
 
 	local playerBalance = getPlayerBalance()
-	local currencyType = PurchaseData.CurrencyType == Enum.CurrencyType.Tix and "tickets" or "robux"
-	local newBalance = playerBalance[currencyType]
+	local newBalance = playerBalance.robux
 
-	if currencyType == "robux" then
-		PostBalanceText.Text = PURCHASE_MSG.BALANCE_NOW..getCurrencyString(PurchaseData.CurrencyType)..formatNumber(newBalance).."."
-	else
-		PostBalanceText.Text = PURCHASE_MSG.BALANCE_NOW..formatNumber(newBalance).." "..getCurrencyString(PurchaseData.CurrencyType).."."
-	end
+	setPostBalanceText(PURCHASE_MSG.BALANCE_NOW..getCurrencyString(PurchaseData.CurrencyType)..formatNumber(newBalance)..".")
```

Comment:
this looks like a place where string.format would make this line infinitely more readable